### DISCUSSION TITLE
human3d: 3D space to original image space translation

### DISCUSSION THREAD

**Discussion by juglaz** [open-mmlab/mmpose#2682] [Q&A]
Hello,

I'm using the human3d model like:
inferencer = MMPoseInferencer(pose3d='human3d')

I'm using the returned 3D keypoints to calculate some reference geometry, which I would like to draw on the original image.
How can I convert the 3D geometry back to the original 2D image space?

Thanks!

**Comment by trenthudgens** [comment]
I'm trying to do the same thing. The closest I found to what I want to do is https://mmpose.readthedocs.io/en/latest/demos.html#d-human-pose-demo but I'm running into issues. I'm messing around with the return_datasample=True argument in in the inferencer, but I run into problem after problem...

using inferencer(show=True) unfortunately uses the 2D keypoints drawn on the original image, instead of the 3D keypoints.
